### DISCUSSION TITLE
How to add a label to a custom bitmap?

### DISCUSSION THREAD

**Discussion by moving-bits** [mapsforge/vtm#1115] [Q&A] [answered]
We display a position indicator on the map (indicator is a custom bitmap, positioned at the current location, and rotated according to current heading).

Is there a best practice for adding a textual label below this indicator (unrotated text)?
(Use case: Elevation information for the position indicated by that indicator)

(Positioning should be close to the bitmap, which does not have a fixed position and can even be outside the current viewport, thus placing the text into a separate View on top of the map view does not help here.)

The map below this indicator can be either online (bitmap) or offline (OpenGL) map.

Thanks for any pointers.

**Comment by devemux86** [comment, marked as answer]
Text can be written on bitmaps used in marker layers, for example see AtlasMultiTextureActivity.

**Reply by moving-bits** [reply]
Ok, thanks for the quick answer. I thought (hoped) there would be an option to draw text directly, but bitmap will be fine, either. At least that way I can use this approach for both VTM and Mapsforge :-)
